Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-edit/user-edit.component.ts

```diff
@@ -66,43 +70,15 @@ export class UserEditComponent {
 
         }, err => {
```

Comment:
The method is for edit user. What happens if edit fails for any reason. You are not providing any feedback to a user that it failed. I think there needs to be a better way to handle the error and report it to client.